imported.xml

```diff
@@ -53,7 +53,16 @@
 		<move file="LICENSE/LICENSE_dual.txt" tofile="LICENSE/LICENSE_MITOrGPLv3WithException.txt" verbose="true">
 		</move>
 	</target>
-
 	
+	<!-- update git submodules -->
+	<target name="updateGitSubmodules" description="Updates (and inits) all git submodules.">
+		<exec executable="git">
+			<arg value="submodule"/>
+			<arg value="update"/>
+			<arg value="--init"/>
+			<arg value="--recursive"/>
+			<arg value="--remote"/>
```

Comment:
@--remote:
ok, yes you are right for release it is maybe better to call it without --remote. but for other cases I think --remote is great because on just developing and starting `ant run` I think we always want to have the current version, do we?
I'll try to change that: on deploy without `--remote`, otherwise with `--remote`

@git-ant-tasks:
I think they only provide basic features like clone or checkout, not things like `git submodule update`.

Reply:
ok, changed the following things:
now buildAll-Relese (and all other builds) only init the git submodules (to commit that is tracked in base repository) and `ant run` also updates the submodules to the latest remote version.

Reply:
hmm:
I like the seperaiton of ini and update.
But I still think it is not a good idea to automatically call updateGitSubmodules in any target, because if you e.g. do ant run it currently updates the submodules and then you try it out and think everything is fine. But when you then call a subsequent ant deploy you get another result.
--> Let's provide the target updateGitSubmodules and use it only manually if a developer wants to manually update the references and commit them.
In all other cases you only want to ensure that the submodule is initialized correctly and pointing to the correct commit.

Reply:
no, in your stated case:
1) `ant run` -> checks out the current HEAD of the submodules
2) `ant deploy` afterwards calls itself only `initGitSubmodules`, which again checks out the latest commited revision in the base repository.
so this should not be the problem.

But if you want, a can nevertheless remove the `updateGitSubmodules` dependency on the run targets.

Reply:
the problem is, that when you do ant run you see the result of the HEAD and maybe think that everything is fine, but then you deploy another revision and maybe even don't recognise that something is not working.
yes, please change it to initGitSubmodules, if it's ok for you.

Reply:
ok, thats right. I changed it and now everything is only dependent on `initGitSubmodules`.

The downside is that now it will happen again, that people are working with old versions of help/WebACS without being aware of it. But yes - we have to communicate this.

So I'll merge this now.